main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"runtime"
+
+	"github.com/alecthomas/kong"
+)
+
+var version string = "v0.0.0"
+
+const description = `
+offscreen turns off/on your Sony Bravia when the screen saver turns on/off
+`
+
+type CLI struct {
+	Version kong.VersionFlag `short:"V" help:"Print program version"`
+}
+
+func main() {
+	// Set maxprocs to 1 - this is a simple program and we don't want
+	// more than one kernel thread for it, even on large boxes.
+	runtime.GOMAXPROCS(1)
```

Comment:
this seems to indicate that go might be doing something different?
just more a curiosity. in general my feeling is less code is better unless there is a good reason.

Reply:
The default as of Go 1.18 is that it defaults to the number of cores/threads on the CPU. My desktop has 8 cores, and my next one may have 16. I recently read of some small programs ending up with a lot of threads when they really didn't need it. In this case, I really don't need it and I want to limit resource usage.

Reply:
Sorry, Go 1.5 not 1.18. It was 2018, not 1.18

Reply:
ok, cool thanks for the info!